cdap-client/src/main/java/co/cask/cdap/client/AuthorizationClient.java

```diff
@@ -223,18 +219,8 @@ private HttpResponse doExecuteRequest(HttpRequest request, int... additionalAllo
     throws IOException, UnauthenticatedException, FeatureDisabledException, UnauthorizedException {
     int[] allowedErrorCodes = new int[additionalAllowedErrorCodes.length + 2];
     System.arraycopy(additionalAllowedErrorCodes, 0, allowedErrorCodes, 0, additionalAllowedErrorCodes.length);
-    allowedErrorCodes[additionalAllowedErrorCodes.length] = HttpURLConnection.HTTP_FORBIDDEN;
-    allowedErrorCodes[additionalAllowedErrorCodes.length + 1] = HttpURLConnection.HTTP_NOT_IMPLEMENTED;
+    allowedErrorCodes[additionalAllowedErrorCodes.length] = HttpURLConnection.HTTP_NOT_IMPLEMENTED;
     HttpResponse response = restClient.execute(request, config.getAccessToken(), allowedErrorCodes);
-    if (HttpURLConnection.HTTP_FORBIDDEN == response.getResponseCode()) {
```

Comment:
Thanks @bdmogal. Can address @anwar6953 's concern. Otherwise PR looks good to me.